.env

```diff
@@ -1,10 +1,27 @@
-PORT=4004
-DB_DRIVER="postgresql"
+SERVER_PORT=4004
+
+# SQLITE Options:
+# DB_DRIVER="sqlite"
 # DB_SQLITE_URL=rego.db
-DB_POSTGRES_HOST="localhost"
-DB_POSTGRES_USERNAME="postgres"
-DB_POSTGRES_PASSWORD="postgres"
-DB_POSTGRES_DB_NAME="postgres"
-DB_POSTGRES_PORT=5432
+
+# POSTGRES Options:
+# DB_DRIVER="postgres"
+# # DB_POSTGRES_DSN = ""  # This overrides other postgres configurations
+# DB_POSTGRES_HOST="localhost"
+# DB_POSTGRES_USERNAME="postgres"
+# DB_POSTGRES_PASSWORD="postgres"
+# DB_POSTGRES_DB_NAME="postgres"
+# DB_POSTGRES_PORT=5432
+
+# SQLITE Options
+DB_DRIVER="mysql"
+# DB_MYSQL_DSN=""  # This overrides other mysqk configurations
+DB_MYSQL_USERNAME="root"
```

Comment:
because the name is literal to the DB I suggest reverting that back to a URL and driver type

Reply:
Will do